assets/css/style.css

```diff
@@ -0,0 +1,17 @@
+.color-cell {
+    height: 100px;
+    width: 100px;
+    padding: 10px;
```

Comment:
for scaling down images please choose either `width` or `height` but not both. otherwise any image with aspect ration different than 1:1 will distort